core/downloadfromdcthread.cpp

```diff
@@ -49,21 +50,38 @@ void DownloadThread::run()
 	qDebug() << "Finishing the thread" << errorText << "dives downloaded" << downloadTable.nr;
 }
 
+static void fill_supported_mobile_list()
+{
+	mobileProductList["Heinrichs Weikamp"] =
+			QStringList({{"OSTC Sport"}, {"OSTC 2N"}, {"OSTC 3"},
+				     {"OSTC 3+"}, {"OSTC 4"}});
+	mobileProductList["Shearwater"] =
+			QStringList({{"Petrel"}, {"Petrel 2"}, {"Perdix"}});
+}
```

Comment:
For iOS it's Petrel 2 and Perdix - but sadly not all Petrel 2 and Perdix (but all Perdix AI) as the earlier Petrel 2 and Perdix models don't support BLE